contracts/AnchoredView/AnchoredView.sol

```diff
@@ -118,58 +115,58 @@ contract AnchoredView is SymbolConfiguration {
 
             address tokenAddress = getCTokenAddress(symbol);
 
-            uint sourcePrice = priceData.getPrice(source, symbol);
+            uint reporterPrice = priceData.getPrice(reporter, symbol);
             uint anchorPrice = getAnchorInUsd(tokenAddress, usdcPrice);
 
             if (tokenAddress == cUsdcAddress || tokenAddress == cUsdtAddress || anchorPrice == 0)  {
```

Comment:
Is this because of fixed prices - I don't think this should check for `cUsdcAddress` or `cUsdtAddress`? Its just a place that can get easily out of sync, now that the config is elsewhere, and this should still return the fixed price anyway? Plus, theoretically if a reporter did want to post those prices, I think it might not be totally unreasonable. Could be in anticipation of a new view coming, or another view, or something

Reply:
it is b/c of fixed prices, good point. There is another issue here that you can only post prices for configured ssets to this view. Maybe there should be an "unconfigured" configuration that writes to data contract, but skips view storage

Reply:
Hmm separate function might be overkill, but where does it err out if its 'unconfigured'? It reverts getting the config which is inside `getAnchorInUsd`?

Reply:
exactly

Reply:
tried it out with the old null-object pattern, I think the "unconfigured" metadata is pretty nice actually